libs/sr-dm/src/dm/relic/RelicSetSkillConfig.ts

```diff
@@ -0,0 +1,39 @@
+import { dumpFile } from '@genshin-optimizer/pipeline'
+import { objFilterKeys, objMap } from '@genshin-optimizer/util'
+import { PROJROOT_PATH } from '../../consts'
+import type { StatDMKey } from '../../mapping'
+import type { RelicSetId } from '../../mapping/relic'
+import { relicSetIdMap } from '../../mapping/relic'
+import { readDMJSON } from '../../util'
+import type { Value } from '../common'
+
+export type RelicSetSkillConfig = {
+  SetID: number
+  RequireNum: number
+  SkillDesc: string
+  PropertyList: Property[]
+  AbilityName: string
+  AbilityParamList: Value[]
+}
+type Property = {
+  PBIJEBOGCKM: StatDMKey // PropertyName
```

Comment:
nit: will probably break in the future once Dim deobfus these fields. I wonder if we can take note of this to find it easier in the future?

Reply:
added comment